src/ipynb/spark_ml.ipynb

```diff
@@ -0,0 +1,681 @@
+{
```

Comment:
<p>To delete the running Dataproc Serverless session, run the following command.</p><p> </p><p>(Do they need to switch kernels in order to do so?)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/terraform-google-analytics-lakehouse/pull/99/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
<p>No it doesn't need to switch kernels. I tested manually and it killed itself.</p>